jsmvc/index.html

```diff
@@ -387,7 +387,7 @@ <h1 class="upper">JavaScript MVC</h1>
 			</div>			
 		</div>
 
-		<script src="../common/js/head.min.js"></script>
+		<script src="../common/lib/js/head.min.js"></script>
```

Comment:
Yup - looks like it matches the demo HTML as well:
https://github.com/hakimel/reveal.js/blob/master/demo.html#L382

Thanks for the details